src/relay/backend/aot_executor_codegen.cc

```diff
@@ -635,6 +601,7 @@ class AOTExecutorCodegen : public ExprVisitor {
           tec::UpdateFunctionMetadata(func, this->function_metadata_);
         });
 
+    function_metadata_.Set(runtime::symbol::tvm_module_main, lowered_module.main_func_info);
```

Comment:
@mbs-octoml I think my point is we should be able to do like `lowered_module.main_module["main"].GetAttr("function_info")`. Maybe your point is to clean this up in future patch after landing this?